deeppavlov/models/bert/bert_classifier.py

```diff
@@ -168,21 +176,14 @@ def _init_optimizer(self):
                                                initializer=tf.constant_initializer(0), trainable=False)
             # default optimizer for Bert is Adam with fixed L2 regularization
             if self.optimizer is None:
-
-                self.train_op = self.get_train_op(self.loss, learning_rate=self.learning_rate_ph,
```

Comment:
check get_train_op function implementation, you lose all features that are implemented in it by using raw optimizer